install-dev/data/xml/tab.xml

```diff
@@ -205,7 +205,7 @@
                     <class_name>AdminThemesCatalog</class_name>
                 </tab>
                 <tab id="Mail_Theme_Parent" id_parent="Look_feel" active="1" enabled="1" hide_host_mode="0" icon="">
-                  <class_name>AdminMailThemeParent</class_name>
+                  <class_name>AdminParentMailTheme</class_name>
```

Comment:
Although I agree this name is better Now that it's been introduced in 176 you can't modify it any more, it creates even more risk of losing some permissions along the way